app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
`Post` model should have method that updates the posts counter for a post. This should be similar to the example below:

```ruby
# app/models/like.rb
...

def update_post_counter
  author.increment!(:posts_counter)
end

```